mozetl/taar/taar_locale.py

```diff
@@ -37,6 +40,12 @@ def store_new_state(source_file_name, s3_dest_file_name, s3_prefix, bucket):
     transfer.upload_file(source_file_name, bucket, key_path)
 
 
+def load_amo_external_whitelist():
+    amo_dump = utils.get_s3_json_content(AMO_DUMP_BUCKET, AMO_DUMP_KEY)
+    white_list_inner = {key: value['guid'] for key, value in amo_dump.items() if value['current_version']['files'][0]['is_webextension']}
```

Comment:
Fly-by: you should really check if any of the files supports webextension, not just the first one. For example:

```
    addon_files = value.get('current_version', {}).get('files', {})
    addon_data['is_webextension'] =\
        any([f.get("is_webextension", False) for f in addon_files])
```